CMakeLists.txt

```diff
@@ -94,33 +94,35 @@ if("${CMAKE_SYSTEM_NAME}" STREQUAL "Linux")
     # Enable relative RPATHs for installed bits
     set (CMAKE_INSTALL_RPATH "\$ORIGIN")
 
-    if(NOT "${CMAKE_CXX_COMPILER_ID}" STREQUAL "Clang")
-        message(FATAL_ERROR "${CMAKE_CXX_COMPILER_ID} is not supported for building!")
+    if(NOT "${CMAKE_CXX_COMPILER_ID}" STREQUAL "GNU")
```

Comment:
This logic is kind of convoluted. Please change it to (pseudocode) `if (clang) { ... } else if (!gnu) {fatal error}`. Thanks.